test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py

```diff
@@ -25,13 +25,13 @@ def step(context, community_name):
     StatusCommunityScreen()
     _statusCommunityScreen.verify_community_name(community_name)
     
-@When("the admin creates a community channel named |any|, with description |any|")
-def step(context, community_channel_name, community_channel_description):
-    _statusCommunityScreen.create_community_channel(community_channel_name, community_channel_description)
+@When("the admin creates a community channel named |any|, with description |any| with the method |any|")
```

Comment:
Another comment related to validation.. maybe we can add some validation regarding the admin user.. Can we verify the community channel is created BY THE ADMIN??

Reply:
I'm not sure. Technically, a user could not create a channel, so we can do another test later where a user joins a community and test if they can create a channel (they shouldn't be able to).